### DISCUSSION TITLE
Repulsive PR Templates

### DISCUSSION THREAD

**Discussion by schmijos** [Homebrew/discussions#1398] [Everyday usage] [answered]
Is it really necessary to throw this at people's faces? It's very repulsive.

![image](https://user-images.githubusercontent.com/245443/116872794-d489f100-ac16-11eb-9184-438dc776546c.png)
[https://github.com/Homebrew/homebrew-core/issues/55625]

Would you mind to adjust the template in a way that the bold top part is only visible to the author of the issue while writing it?

**Comment by SMillerDev** [comment, marked as answer]
This used to be a comment above the issue template that the template mentioned to remove. Unfortunately not everyone reads the issue template (guess why we added the message) and therefore it remained in a lot of issues. (and it got boldened because `# comment` is an indication of a header in markdown).

Luckily we've switched to issue forms a couple of months ago so only the author gets this "repulsive" message.

**Comment by carlocab** [comment]
Did you check if this message still appears in recent issues?

**Reply by schmijos** [reply]
Good point. I did not, because I currently don't have an issue.

**Reply by carlocab** [reply]
I removed the header from the issue in your screenshot so you don't have to see it anymore while reading that particular one.

**Comment by schmijos** [comment]
I guess I was a bit too fast on the keyboard. Recent issues seem presented fine to the public.
